README.md

```diff
@@ -83,7 +83,10 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 #### Text Editors
 
 * [Atom](https://atom.io) - Hackable text editor for the 21st century made by GitHub. [Atom Plugins](editor-plugin.md#atom-plugin). [![Open-Source Software][OSS Icon]](https://github.com/atom/atom) ![Freeware][Freeware Icon]
+* [BBEdit](http://www.barebones.com/products/bbedit/) - The leading professional HTML and text editor for Mac.
```

Comment:
I've never actually seen this in the wild, and I think Brackets and WebStorm sorta cover this category. Maybe remove BBEdit?

Reply:
@alichtman BBEdit is not very powerful?

Reply:
@alichtman BBEdit used to be very powerful, I like it very much.

Reply:
https://www.barebones.com/products/bbedit/bbedit12.html

Reply:
@alichtman 👌

Reply:
@heldev You should remove BBEdit